thriftrw/spec/enum.pyx

```diff
@@ -231,6 +232,7 @@ def enum_cls(enum_spec, scope):
     enum_dct['__doc__'] = enum_docstring(
         enum_spec.name, enum_spec.items.items()
     )
+    enum_dct['__hash__'] = lambda self: hash(values)
```

Comment:
To be clear, enum objects themselves aren't hashable.

The generated class is roughly equivalent to,

```
class MyEnum(object):
    ITEM_1 = 1
    ITEM_2 = 2
```

So `__hash__` on the object is never called because `MyEnum()` is never used.

That said, I suppose there's no harm in keeping this.